apps/perf-test-react-components/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
```

Comment:
it would be great to follow (leverage new DX) what `vr-` test decoupling started

adding

```
"extends": "../../tsconfig.base.json",
```

Reply:
Added this to both v8 and v9 perf test packages